docs/source/api/exporters.rst

```diff
@@ -65,22 +65,6 @@ These functions are essentially convenience functions that
 wrap the functionality of the classes documented in the previous
 section.
```

Comment:
@michaelpacer this description should also be updated along with the change in the things it refers to!